mlflow/gateway/schemas/chat.py

```diff
@@ -24,72 +22,56 @@ class RequestPayload(BaseRequestPayload):
 
     class Config:
         schema_extra = {
-            "example": {
-                "messages": [
-                    {
-                        "role": "user",
-                        "content": "hello world",
-                    }
-                ],
-                "temperature": 0.0,
-                "max_tokens": 64,
-                "stop": ["END"],
-                "candidate_count": 1,
-            }
+            "messages": [
+                {"role": "system", "content": "You are a helpful assistant."},
+                {"role": "user", "content": "Hello!"},
+            ],
+            "temperature": 0.0,
+            "max_tokens": 64,
+            "stop": ["END"],
+            "n": 1,
         }
 
 
-class FinishReason(str, Enum):
-    STOP = "stop"
-    LENGTH = "length"
-
-
```

Comment:
OpenAI and other vendors have added a lot of finish reasons lately. I think it's best to leave this as a string for now and potentially figure out how to standardize later, if needed

Reply:
+1. This will be whack-a-mole otherwise

Reply:
+1